framework/include/mfem/functormaterials/MFEMGenericFunctorVectorMaterial.h

```diff
@@ -19,7 +19,7 @@ class MFEMGenericFunctorVectorMaterial : public MFEMFunctorMaterial
 
 protected:
   const std::vector<std::string> & _prop_names;
-  const std::vector<MFEMVectorCoefficientName> & _prop_values;
+  const std::vector<MFEMVectorCoefficientName> _prop_values;
```

Comment:
I'd revert this change. No reason to store this twice, once in the input parameters warehouse and once here

Reply:
Oops I was looking at the wrong local repo. I see that this does indeed have to be its own storage